doc/classes/TileMap.xml

```diff
@@ -198,6 +199,8 @@
 			<argument index="1" name="end" type="Vector2" default="Vector2( 0, 0 )">
 			</argument>
 			<description>
+				Apply autotiling rules for the cells in given region (specified by grid-base X and Y coordinates).
```

Comment:
- Apply -> Applies
- for -> to
- in given region -> in the given region
- grid-base -> grid-based